portfolio/src/main/java/com/google/sps/servlets/LikeServlet.java

```diff
@@ -36,33 +36,45 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Key likeKey = KeyFactory.createKey("Comment", commentId);
```

Comment:
Nit: This is a commentKey not a likeKey.

Reply:
Done